keras/utils/test_utils.py

```diff
@@ -88,6 +88,7 @@ def layer_test(layer_cls, kwargs={}, input_shape=None, input_dtype=None,
     # check shape inference
     model = Model(x, y)
     expected_output_shape = layer.compute_output_shape(input_shape)
+    model.compile('rmsprop', 'mse')
```

Comment:
Will this change break other Backend like TF or CNTK?

Reply:
No.